### PR TITLE
added modtopia and added manifest.json

### PR DESCRIPTION
**Pull request by wchtig**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/modtopia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "server_name": "modtopia",
+  "nice_name": "Modtopia",
+  "direct_ip": "modtopia.de",
+  "server_wildcards": [
+    "%.modtopia.de"
```
Comment (DoJapHD) [inline minecraft_servers/modtopia/manifest.json:6]: Not a real wildcard
https://i.imgur.com/mwjhkFi.png

File: minecraft_servers/modtopia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  ],
+  "social": {
+    "web_shop": "https://shop.modtopia.de",
+    "youtube": "https://youtube.modtopia.de",
```
Comment (DoJapHD) [inline minecraft_servers/modtopia/manifest.json:10]: Has to be "https://www.youtube.com/channel/[CHANNELID]"

File: minecraft_servers/modtopia/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "tiktok": "modtopia",
+    "instagram": "modtopia.social",
+    "discord": "https://discord.modtopia.de"
```
Comment (DoJapHD) [inline minecraft_servers/modtopia/manifest.json:13]: Reserved for partners; use https://discord.gg/[INVITE]

**Review by DoJapHD — CHANGES_REQUESTED** [review]
icon.png and icon@2x.png are missing but are required!

**Comment by wchtig** [comment]
> icon.png and [icon@2x.png](mailto:icon@2x.png) are missing but are required!

how can i add this?

File: minecraft_servers/modtopia/background.png
Comment (DoJapHD) [inline minecraft_servers/modtopia/background.png:1]: Compress with e.g. https://tinypng.com

File: minecraft_servers/modtopia/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/modtopia/background@2x.png:1]: Compress with e.g. https://tinypng.com

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**